editor.planx.uk/src/@planx/components/Feedback/Public/Public.tsx

```diff
@@ -83,15 +84,17 @@ const FeedbackComponent = (props: PublicProps<Feedback>): FCReturn => {
         id={"DESCRIPTION_TEXT"}
         openLinksOnNewTab
       />
-      <Box pt={2} mb={3}>
+      <Box my={4}>
         {props.ratingQuestion && (
           <InputLabel
             label={
-              <ReactMarkdownOrHtml
-                source={props.ratingQuestion}
-                id={"RATING_QUESTION"}
-                openLinksOnNewTab
-              />
+              <Typography component="span" fontWeight="700">
+                <ReactMarkdownOrHtml
+                  source={props.ratingQuestion}
+                  id={"RATING_QUESTION"}
+                  openLinksOnNewTab
+                />
+              </Typography>
```

Comment:
I think this will produce invalid HTML.

`ReactMarkdownOrHTML` serves content from a rich text input and could contain a number of elements such as a `h1` or `h2`, leading to a structure like this - 

```html
<span>
   <div>
     <h1></h1>
  </div>
</span>
```

Another example - 

<img width="365" alt="image" src="https://github.com/user-attachments/assets/d8ffd569-5a99-474b-aa0a-3931180b4c0b">

A few options in terms of resolving this - we could change the `component` prop here to `div` (if allowed by `Typography` ?), or we could change the EditorModal to actually make this just a text field and not a rich text field.

I'd lean towards making this just a text input, but it's worth checking with @jamdelion and the content team if there's a valid reason to make this rich text 👍

Reply:
I've removed the `<Typography>` wrapper and instead wrapped the `defaultContent` for these labels with `<strong>` tags so that if we keep the rich text editor there will be a styled default.

Agree on the preference for text input for input labels 👍